src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -64,6 +64,7 @@ public class Feature
         private static readonly Feature __mmapV1StorageEngine = new Feature("MmapV1StorageEngine", new SemanticVersion(0, 0, 0), new SemanticVersion(4, 1, 0, ""));
         private static readonly Feature __partialIndexes = new Feature("PartialIndexes", new SemanticVersion(3, 2, 0));
         private static readonly ReadConcernFeature __readConcern = new ReadConcernFeature("ReadConcern", new SemanticVersion(3, 2, 0));
+        private static readonly Feature __retryableReads = new Feature("RetryableReads", new SemanticVersion(4, 0, 0)); // should be 4.2.0
```

Comment:
Why not 3.6?

https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#supported-server-versions